README.md

```diff
@@ -484,7 +491,11 @@ assuming your kernel is configured to work that way.
 u-root can create an initramfs in two different modes, specified by `-build`:
 
 *   `bb`/`gbb` mode: One busybox-like binary comprising all the Go tools you ask to
-    include. See [here for how it works](pkg/bb/README.md).
+    include, which can be [templates](#templates) and/or individual commands.
```

Comment:
makes it sound like templates are only possible with gbb/bb mode -- they work with any mode

Reply:
gotcha - I'll drop that